code/modules/vore/eating/vorepanel_vr.dm

```diff
@@ -1119,25 +1119,25 @@
 				if("10 minutes")
 					host.vore_selected.gen_time = 0
 				if("30 minutes")
-					host.vore_selected.gen_time = 2
+					host.vore_selected.gen_time = 4
 				if("1 hour")
-					host.vore_selected.gen_time = 5
+					host.vore_selected.gen_time = 10
 				if("3 hours")
-					host.vore_selected.gen_time = 17
+					host.vore_selected.gen_time = 34
 				if("6 hours")
-					host.vore_selected.gen_time = 35
+					host.vore_selected.gen_time = 70
 				if("12 hours")
-					host.vore_selected.gen_time = 71
+					host.vore_selected.gen_time = 142
 				if("24 hours")
-					host.vore_selected.gen_time = 143
+					host.vore_selected.gen_time = 286
```

Comment:
Doubling these will only double the amount of time it takes to produce a single unit and that would mess things up.